bridge/github/config.go

```diff
@@ -184,6 +194,135 @@ func randomFingerprint() string {
 	return string(b)
 }
 
+func promptTokenOptions(owner, project string) (string, error) {
+	for {
+		fmt.Println()
+		fmt.Println("[0]: user provided token")
```

Comment:
All the other prompt starts at 1, but this one start at 0.

Reply:
Done